index.md

```diff
@@ -1,144 +1,51 @@
+|![area](https://img.shields.io/badge/Graphs-yellowgreen) &nbsp; ![technique](https://img.shields.io/badge/Hypergraphs-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[Transforming Healthcare Data with Graph-based Techniques - Next Steps](https://nhsx.github.io/nhsx-internship-projects/transforming-healthcare-data-graph-based-sail-update/)|
+|June 2023|
+|![area](https://img.shields.io/badge/Graphs-yellowgreen) &nbsp; ![technique](https://img.shields.io/badge/GNNs-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModalData-blueviolet)|[Exploring Data Representations - Graph Neural Networks](https://nhsx.github.io/nhsx-internship-projects/exploring-data-representations-gnn/)|
+|![area](https://img.shields.io/badge/Simulation-darkblue) &nbsp; ![technique](https://img.shields.io/badge/ProcessMining-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[Process Mining to Generate Healthcare Pathways](https://nhsx.github.io/nhsx-internship-projects/process-mining-generate-pathways/)|
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/Memorisation-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModalData-blueviolet)|[Understanding Privacy Leakage when using Language Models in Healthcare](https://nhsx.github.io/nhsx-internship-projects/language-model-privacy-leakage/)|
+
+<ul class="nhsuk-grid-row nhsuk-card-group">
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+    <div class="nhsuk-card nhsuk-card--clickable">
+        <img class="nhsuk-card__img" src="assets/img/graph.jpg" alt="teaser" />
+        <div class="nhsuk-card__content">
+            <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="projects.html">Projects</a>
+            </h2>
+            <p class="nhsuk-card__description">List of active and open projects to either get involved with or apply to in our next round</p>
+        </div>
+    </div>
+  </li>    
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">  
+    <div class="nhsuk-card nhsuk-card--clickable">
+        <img class="nhsuk-card__img" src="assets/img/genetic-algo.jpg" alt="teaser" />
+        <div class="nhsuk-card__content">
+            <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="overview.html">Scheme Overview</a>
+            </h2>
+            <p class="nhsuk-card__description">Details of aims and application process of the scheme</p>
+        </div>
+    </div>
+  </li>
+</ul>
+
+Please get in contact by emailing **england.tdau@nhs.net** with further questions around the internship.  
```

Comment:
Needs updating to be hyperlink (currently just bold). Also confirm this is the correct email address at the moment.

Reply:
add form of mailto:

Reply:
added